src/VMware/VMware.Autorest/custom/Get-AzVMwareAddon.ps1

```diff
@@ -72,7 +72,7 @@ INPUTOBJECT <IVMwareIdentity>: Identity Parameter
 https://learn.microsoft.com/powershell/module/az.vmware/get-azvmwareaddon
 #>
 function Get-AzVMwareAddon {
-[OutputType([Microsoft.Azure.PowerShell.Cmdlets.VMware.Models.Api20211201.IAddon])]
+[OutputType([Microsoft.Azure.PowerShell.Cmdlets.VMware.Models.Api20230301.IAddon])]
```

Comment:
The namespace doesn't contains API version any more.

Reply:
Fixed this in the latest build. It should not contain any api version.